### DISCUSSION TITLE
GLFW error #6554: GLX: Failed to create context: GLXBadFBConfig

### DISCUSSION THREAD

**Discussion by DRosemei** [NVlabs/instant-ngp#499] [Q&A]
Errors are below
![image](https://user-images.githubusercontent.com/27892562/164130424-670144b4-dfa1-4473-80f0-064d1d5bcce3.png)
And I run 'glxinfo | grep "OpenGL version"', it gots '-bash: glxinfo: command not found
' Does it mean I need to install opengl in centos?

**Comment by DRosemei** [comment]
When I run 'python scripts/run.py  --mode=nerf  --scene=./data/nerf/fox --save_mesh=./fox.obj --gui'
get 
![image](https://user-images.githubusercontent.com/27892562/164146997-7bee5eb9-0f69-438c-b9ec-25eb55ab6c01.png)

**Comment by Tom94** [comment]
Something fishy seems to indeed be the case with your OpenGL setup. It _is_ already installed -- otherwise the program wouldn't have compiled and started successfully -- so I recommend searching the web for why the "GLXBadFBConfig" could occur on your system. My guess without googling is as good as yours on this one, unfortunately.

That said, if you don't actually need the GUI, both run.py and the testbed binary offer command line options to run without GUI.

**Comment by Harperrrr111** [comment]
@DRosemei  
Hi, have you addressed this issue? I met the same question.
